### ISSUE TITLE
Show jid or nickname next to window number in status bar

### ISSUE THREAD

**Issue by asmartin** [boothj5/profanity#893]
I recently switched from finch to profanity and am very impressed with how polished profanity is! One thing that would be helpful is to be able to know which username is associated with each window on the status bar, rather than just seeing the numbers. Would it be possible to display the beginning of the jid or nickname, e.g. something like this:

```
[1 - john] [2 - steve] [3 - dave] ...
```

You could even configure how many characters are displayed, so if you are only a small screen and only want 2 characters displayed, configure that and then it would show up in a more compact form:

```
[1 - jo] [2 - st] [3 - da] ...
```

Is this possible?

**Comment by boothj5** [comment]
Closing this issue, but linking to #416 since it's very similar.

I've also added the `0.6.0` milestone as a few people have asked for it, so it should be in the next major release.

**Comment by asmartin** [comment]
Great, thanks!

**Comment by boothj5** [comment]
This has now been implemented in #416
